fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -465,6 +506,11 @@ struct TenantManagementWorkload : TestWorkload {
 					ASSERT(entry.get().tenantGroup == tenantItr->second.tenantGroup);
 					ASSERT(entry.get().tenantState == TenantState::READY);
 
+					if (operationType != OperationType::SPECIAL_KEYS) {
+						Versionstamp currentVersionstamp = wait(getLastTenantModification(self, operationType));
+						ASSERT_GT(currentVersionstamp.version, originalVersionstamp.version);
```

Comment:
A slightly stronger property we could check is that the modification versionstamp has a version larger than a cluster version before the operation (the cluster version should be larger than the last tenant modification versionstamp). I don't feel super strongly that we need to do that, though.

Reply:
I'll add this check